hack/release/prepare-changelog.sh

```diff
@@ -117,7 +117,23 @@ echo "### Changed"
 echo ""
 # Put the changes under "Changed" first, release manager needs to move them to appropriate sections manually.
 gh pr list -s merged -B ${branch} --search "merged:>${release_start_time} sort:updated-desc label:action/release-note" -L $limit --json number,title,author,labels --template \
-'{{range .}}{{tablerow (printf "- %s. ([#%v](https://github.com/antrea-io/antrea/pull/%v), [@%s])" .title .number .number .author.login)}}{{end}}'
+'{{range .}}{{tablerow (printf "- %s. ([#%v](https://github.com/antrea-io/antrea/pull/%v), [@%s])" .title .number .number .author.login)}}{{end}}' | cat
+
+if [ "$patch_number" != "0" ]; then
```

Comment:
Doesn't it generate each PR twice? 

Maybe we could do this:
1. Add "-label:kind/cherry-pick" to the above search so it finds original PRs only and their changelog can be generated directly
2. Add a "label:kind/cherry-pick" search like below so it finds cherry-picked PRs and their changelog should be generated based on their original PRs

I think this applies to both minor releases and patch releases as it sometimes happened that a PR is merged to a release branch then cherry-picked to main branch

Reply:
It will not generate PR twice as the cherry-picked PR does not have the label `action/release-note`. Do you think we should remove the patch number checking in line 122 so that it will work for minor releases as well?